src/core_modules/capture-core/components/WorkingLists/TeiWorkingLists/epics/teiViewEpics/helpers/getEventListData/getEventListData.js

```diff
@@ -54,7 +55,7 @@ const createApiTEIsQueryArgs =
     pageSize,
     trackedEntity: trackedEntityIds,
     fields:
-    'trackedEntity,createdAt,orgUnit,attributes[attribute,value],enrollments[enrollment,status,orgUnit,enrolledAt]',
+    'trackedEntities,createdAt,orgUnits,attributes[attribute,value],enrollments[enrollment,status,orgUnit,enrolledAt]',
```

Comment:
Hi @henrikmv,
Currently, the program stage WL is giving a 400 error. The plural renaming should be applied to the query parameter, not the requested fields. I think something like this will work: 

```
const createApiTEIsQueryArgs =
({ pageSize, programId: program }, trackedEntityIds): { [string]: any } => {
    const filterQueryParam: string = featureAvailable(FEATURES.newEntityFilterQueryParam)
        ? 'trackedEntities'
        : 'trackedEntity';

    return {
        program,
        pageSize,
        [filterQueryParam]: trackedEntityIds,
        fields:
        'trackedEntity,createdAt,orgUnit,attributes[attribute,value],enrollments[enrollment,status,orgUnit,enrolledAt]',
    };
};
```


Could you have a look? Thanks!